containers/java-postgres/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+{
+	"name": "Java & PostgreSQL",
+	"dockerComposeFile": "docker-compose.yml",
+	"service": "app",
+	"workspaceFolder": "/workspace",
+	
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"sqltools.connections": [
+			{
+			  "name": "Container database",
+			  "driver": "PostgreSQL",
+			  "previewLimit": 50,
+			  "server": "localhost",
+			  "port": 5432,
+			  // NOTE: database/username/password should match values in docker.compose.yml
+			  "database": "postgres",
+			  "username": "postgres",
+			  "password": "postgres"
```

Comment:
hardcoding a SQL password feels wrong. The `dotnet-mssql` passes an argument to the `Dockerfile`. Let's follow that pattern.

Reply:
hi @xtophs, I believe we already follow what both `python-3-postgres` and `dotnet-mssql` do -- the "db password" is present in both the `devcontainer.json` and in the `docker-compose.yml` files

Reply:
+1 This is the existing pattern in almost all the other files as well.

Reply:
sync'ing more offline -- the password only lives in the `devcontainer.json` and `docker-compose.yml` now, so is no longer in the java code.

Reply:
I believe this has been declared to be out of scope for now, unless I'm mistaken? Do we think this can be resolved @dnastrain  and @xtophs ?